packages/web/components/denom/denom-image.tsx

```diff
@@ -0,0 +1,29 @@
+import { FunctionComponent } from "react";
+import Image from "next/image";
+
+import { DenomImageProps } from "./types";
+
+export const DenomImage: FunctionComponent<DenomImageProps> = ({
+  denom,
+  size = 24,
+}) => {
+  return (
```

Comment:
Nitpick: if there's no content in body of function I try to return a single expression to simplify code (the content of the `return (...)` statement)